src/community/features-templating/features-templating-core/src/test/resources/org/geoserver/featurestemplating/response/GeoJSONQueryablesSchema.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "MappedFeatureGeoJSON Queryables Override",
+  "type": "object",
+  "properties": {
+    "testPropertyForSchemaOverrideGeoJSON": {
+      "type": "string",
+      "title": "Test Property for GeoJSON Schema Override"
+    }
+    // ...add more properties as needed for testing...
```

Comment:
This is not even valid JSON any more (JSON does not have comments). 
Is this file actually used? If so, is the parser setup to allow comments?
E.g., [Jackson can be configured to allow java style comments](https://github.com/FasterXML/jackson-core/wiki/JsonReadFeatures#general) but I don't see it being setup such way here.